WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/DashboardGoogleDomainsCardView.swift

```diff
@@ -1,13 +1,22 @@
 import SwiftUI
 
 struct DashboardGoogleDomainsCardView: View {
+    private var buttonClosure: () -> ()
+
+    init(buttonClosure: @escaping () -> ()) {
+        self.buttonClosure = buttonClosure
+    }
```

Comment:
💡 This is a minor suggestion. Personally, I'd prefer to name this property `action` or `buttonAction`. This naming convention can be found in UIKit and SwiftUI too. For example, in `SwiftUI.Button` initializer, the param is named `action`.